doc/ref_model/chapters/chapter03.rst

```diff
@@ -357,10 +357,94 @@ The hardware infrastructure manager needs to support the following functional as
 - **Additional Management Functions**: include software and configuration life cycle management, identity management,
   access management, policy management (e.g. to enforce security policies), etc.
 
-Left for future use
--------------------
+The Redfish Standard and the ODIM Implementation 
```

Comment:
Can you please clarify the purpose of introducing Redfish and ODIM here? Is this a for information, kinda information sharing or the purpose is to mandate Redfish or ODIM in RM?

Reply:
The Introduction in the reference model is informational. Redfish provides a standard interface that can manage compute/network/storage devices. 
ODIM implements and exposes a single redfish API. ODIM can also help manage non redfish capable devices by way plugins (ACI fabric adapter being an example). We think it could be a good fit for the role of 'Hardware Infrastructure Manager' in Anuket reference implementation.

Reply:
Further this is not a mandatory component of the Reference Implementation, but is an optional component.